Tests/External/Plugins/ASCompletion.Tests/Completion/ASGeneratorTests.cs

```diff
@@ -3373,7 +3373,7 @@ internal static string HaxeImpl(ScintillaControl sci, string fileName, Generator
                     var sourceText = ReadAllTextHaxe(fileName);
                     fileName = GetFullPathHaxe(fileName);
                     fileName = Path.GetFileNameWithoutExtension(fileName).Replace('.', Path.DirectorySeparatorChar) + Path.GetExtension(fileName);
-                    fileName = Path.GetFullPath(fileName);
+                    fileName = Path.Combine(PathHelper.AppDir, fileName);
```

Comment:
Same problem as with Scintilla. While upgrading to NUnit 3 my startup directory was not the same as FD so this logic was not running properly.